src/main/java/com/foundation/search/view/ParametersPanel.java

```diff
@@ -65,53 +91,210 @@ private void setting() {
      * to this Panel.
      */
     private void init() {
-        //Initializing the components
-        instruction = new JLabel("Please enter the parameters to perform the search");
-        textFilesFound = new JLabel("All files found according to the criteria set:");
-        nameLabel = new JLabel("Name:");
+        //Creating the labels
+        JLabel pathLabel = new JLabel("    Location (Mandatory):");
+        JLabel nameLabel = new JLabel("    By Name:");
+        JLabel extensionLabel = new JLabel("    By Extension:");
+        JLabel sizeLabel = new JLabel("    By Size: ");
+        JLabel ownerLabel = new JLabel("    By owner: ");
+        JLabel contentLabel = new JLabel("    By content(Only Files): ");
+        //defining the browse button
+        pathInputLabel = new JTextField();
+        btnBrowse = new JButton("Browse");
+        pathInput = new JFileChooser();
+        btnBrowse.addActionListener(new ActionListener() {
```

Comment:
You can try to move this ActionListener into a separate method and call it using a lambda expression.
If you don't have time now, You can work on it on your next PR round.

Reply:
Yes, it will be refactorized in next version.
Thanks